app/main.jsx

```diff
@@ -24,9 +26,9 @@ const ExampleApp = connect(
 render (
   <Provider store={store}>
     <Router history={browserHistory}>
-      <Route path="/" component={ExampleApp}>
-        <IndexRedirect to="/jokes" />
-        <Route path="/jokes" component={Jokes} />
+      <Route path="/" component={Root}>
+        <IndexRedirect to="/potions"/>
+        <Route path="/potions" component={Navbar}/>
```

Comment:
I think it would make more sense to put the navbar component inside the ExampleApp component because that is what is going to render on every page. We should keep index redirect the same as how we currently have it on master, because that is how we render all the products on our page:

` <Provider store={store}>
    <Router history={browserHistory}>
      <Route path="/" component={ExampleApp} onEnter={onLoadProducts}>
        <IndexRedirect to="/home" />
        <Route path="/home" component={ProductsContainer} />
      </Route>
    </Router>
  </Provider>,
  document.getElementById('main')`